server/integration/versions/pom.xml

```diff
@@ -17,11 +17,9 @@
    <description>Infinispan Server - BOM</description>
 
    <properties>
-      <product.version>${project.version}</product.version>
-      <product.name>infinispan-server</product.name>
       <maven.compiler.target>1.8</maven.compiler.target>
       <maven.compiler.source>1.8</maven.compiler.source>
-      <server.output.dir.prefix>infinispan-server</server.output.dir.prefix>
+      <server.output.dir.prefix>${infinispan.brand.prefix}-server</server.output.dir.prefix>
```

Comment:
Just thinking - maybe it would be better to leave it as `infinispan-server`. This way we could spare a couple of renames...